File: Java/073. RaioPositivo/RaioPositivo.java
import java.util.Scanner;
public class RaioPositivo
{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        double raio = 0.0;
        while (raio <= 0.0)
        {
            System.out.print("Informe o raio: ");
            raio = scan.nextDouble();
        }        
        System.out.printf("A area eh: %.2f\n", 3.1415 * raio * raio);
        scan.close();
    }
}

File: C++/073. RaioPositivo/RaioPositivo.cpp
#include <iostream>
#include <iomanip>
int main() {
    double raio = 0;
    while (raio <= 0)
    {
        std::cout << "Informe o raio: ";
        std::cin >> raio;
    }
    std::cout << std::fixed << std::setprecision(2);  
    std::cout << "A area eh: " << 3.1415 * raio * raio << std::endl;
    return 0;
}